pkg/platform/abstract/platform.go

```diff
@@ -1867,6 +1868,11 @@ func (ap *Platform) enrichEnvVars(config *functionconfig.Config) {
 					config.Spec.Env = append(config.Spec.Env, newEnvVar)
 				}
 			}
+			// If EnvFrom is set in the platform config, add the EnvFrom object at the beginning of the list of EnvFrom in the function config.
+			// We add it at the beginning so that the values in the function config take priority over those in the platform config.
+			if !reflect.DeepEqual(ap.Config.Runtime.Common.EnvFrom, v1.EnvFromSource{}) {
```

Comment:
Is this better than checking `nil` and `len`?
I would add a `len` check anyways, so we could skip the `append` if it not defined at all.

Reply:
@TomerShor it is not possible to check either `ap.Config.Runtime.Common.EnvFrom` is nil or len()==0 since it is an obj of type `v1.EnvFromSource`. But since we are going to change it to list of `v1.EnvFromSource`, it will be fine